setup.py

```diff
@@ -16,7 +16,7 @@
 
 setup(
     name="pro-wes",
-    version=__version__,  # noqa: F821  # pylint: disable=undefined-variable
+    version=__version__,  # type: ignore # noqa: E501 F821 # pylint: disable=undefined-variable
```

Comment:
:rofl:

Reply:
I have no idea what to do here :slightly_smiling_face:

Reply:
It's fine, just funny :)

Actually, I think we don't run mypy on `setup.py` in other repos. But you can leave it. Soon we will migrate to `pyproject.toml` anyway and then that will not be an issue anymore.